pages/workspaces/[workspaceId].tsx

```diff
@@ -1,20 +1,23 @@
-import { Heading } from '@chakra-ui/react';
 import WorkspacesLayout from '@/layouts/Workspaces';
 import withAuthentication from '@/hoc/withAuthentication';
 import { useRouter } from 'next/router';
 import { useWorkspaceQuery } from '@/graphql/workspace/QueryWorkspace.generated';
 
+import WorkspaceHeading from './components/WorkspaceHeading';
+
 const Workspaces: React.FC = () => {
   const router = useRouter();
   const workspaceId = router.query.workspaceId as string;
 
-  const { data } = useWorkspaceQuery({ variables: { workspaceId } });
-
-  console.log({ data });
+  const { data, loading } = useWorkspaceQuery({ variables: { workspaceId } });
 
   return (
     <WorkspacesLayout>
-      <Heading as="h1">Manage Workspaces</Heading>
+      <WorkspaceHeading
+        workspaceName={data?.workspace?.workspaceName}
```

Comment:
Do you think we should update the schema to say this query always returns a workspace? If there is  a place within the resolver where it can fail or not find one I would think it would be error related. But I think the return type of the query should be `Workspace!`